test/master-playlist-controller.test.js

```diff
@@ -19,6 +19,40 @@ import { Hls } from '../src/videojs-contrib-hls';
 /* eslint-enable no-unused-vars */
 import Playlist from '../src/playlist';
 
+const generateMedia = function(isMaat, isMuxed, hasVideoCodec, hasAudioCodec) {
+  const codec = (hasVideoCodec ? 'avc1.4d400d' : '') +
```

Comment:
Would be good to use different codecs from the defaults when testing provided codecs